molgenis-app/src/main/java/org/molgenis/app/WebAppPermissionRegistry.java

```diff
@@ -82,6 +82,7 @@
 		register(PLUGIN, ImportWizardController.ID, editor, READ);
 		register(ENTITY_TYPE, FREEMARKER_TEMPLATE, anonymousUser, READ);
 		register(ENTITY_TYPE, FREEMARKER_TEMPLATE, user, READ);
+		register(ENTITY_TYPE, FREEMARKER_TEMPLATE, editor, WRITE);
```

Comment:
Will registering this permission also grant WRITE permission on FREEMARKER_TEMPLATE to the manager role?

Reply:
Discussed with @tommydeboer and @sidohaakma: yes, because the editor role includes the manager role